db/local/db.py

```diff
@@ -0,0 +1,173 @@
+import argparse
+import os
+import subprocess
+from typing import Type
+
+from peewee_migrate import Router
+
+from src.base_model import BaseModel, db
+
+MIGRATION_DIRECTORY: str = "db/local/migrations"
+
+models: list[Type[BaseModel]] = BaseModel.__subclasses__()
+
+
+def run(args: argparse.Namespace):
+    """Run the database
+
+    :param args: parsed arguments containing the string `database_name`
+    :type args: argparse.Namespace
+    """
+    name = args.database_name
+    print("Running database...")
+    print(os.getcwd())
+    subprocess.run(
+        [
+            "docker-compose",
+            "-f",
+            "db/local/docker-compose.yml",
+            "up",
+            "-d",
+            "-".join(["postgres", name]),
+        ],
+        check=True,
+    )
+
+
+def stop(args: argparse.Namespace):
+    """Stop the database
+
+    :param args: parsed arguments containing the string `database_name`
+    :type args: argparse.Namespace
+    """
+    name = args.database_name
+    print("Stopping database...")
+    subprocess.run(
+        [
+            "docker-compose",
```

Comment:
This also assumes that you are member of the `docker` group (linux), as otherwise you will not have access to docker without `sudo`

Reply:
Good catch too! We should add this to the documentation, that you should be part of the docker group.